src/test/java/org/assertj/core/api/charsequence/CharSequenceAssert_isEqualToNormalizingPunctuation_Test.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api.charsequence;
+
+import org.assertj.core.api.CharSequenceAssert;
+import org.assertj.core.api.CharSequenceAssertBaseTest;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Created by harisha talanki on 2/29/20
+ */
+public class CharSequenceAssert_isEqualToNormalizingPunctuation_Test extends CharSequenceAssertBaseTest {
+
+  @Override
+  protected CharSequenceAssert invoke_api_method() {
+    return assertions.isEqualToNormalizingPunctuation("Game of Thrones");
+  }
+
+  @Override
+  protected void verify_internal_effects() {
+    verify(strings).assertEqualsNormalizingWPunctuation(getInfo(assertions), getActual(assertions), "Game of Thrones");
+  }
+
+  @Test
+  void should_fail_if_expected_string_is_empty() {
```

Comment:
all the tests below and this one should be in `Strings` and testing `assertEqualsNormalizingPunctuation`.
the tests should check that the error is built with ShouldBeEqualNormalizingPunctuation

https://github.com/joel-costigliola/assertj-core/blob/master/src/test/java/org/assertj/core/internal/throwables/Throwables_assertHasRootCauseMessage_Test.java is a good example to follow.

Consider writing parameterized tests to avoid too much repetition (see https://github.com/joel-costigliola/assertj-core/blob/0afd84c5c6f30472d19869d169fca0bd80c9dc1c/src/test/java/org/assertj/core/internal/strings/Strings_assertContainsWhitespaces_Test.java or https://github.com/joel-costigliola/assertj-core/blob/0afd84c5c6f30472d19869d169fca0bd80c9dc1c/src/test/java/org/assertj/core/util/FloatComparatorTest.java). 

what we should only be doing in this test class is to verify that we delegate to Strings correctly.